packages/mui-base/src/Menu/Menu.test.tsx

```diff
@@ -260,7 +260,7 @@ describe('<Menu />', () => {
       const item1 = getByTestId('item-1');
       const item2 = getByTestId('item-2');
 
-      await act(() => {
+      await act(async () => {
```

Comment:
why async is needed here if 
items[0].focus(); does not have await ? 
it make sense to have it as await act(async () => items[0].focus());
or await act(async () => {
await items[0].focus();
})

Reply:
From https://react.dev/reference/react/act#await-act-async-actfn

> We recommend using `act` with `await` and an `async` function. Although the sync version works in many cases, it doesn’t work in all cases and due to the way React schedules updates internally, it’s difficult to predict when you can use the sync version.
>
> We will deprecate and remove the sync version in the future.

or, it changes the behavior of `act`. I understand it looks a bit strange to not `await` anything. but it's nicer than  doing

```tsx
await act(() => {
  items[0].focus();
  return Promise.resolve();
})
```

No need to await `items[0].focus()`. It doesn't return a promise.